helm/alfresco-content-services/values.yaml

```diff
@@ -534,7 +534,7 @@ postgresql-syncservice:
   enabled: true
   replicaCount: 1
   nameOverride: postgresql-syncservice
-  imageTag: "11.7.0"
+  imageTag: "13.1"
   commonAnnotations:
```

Comment:
Not sure this is correct, as I would not have expected 11.7.0 as the original value. Community used just 11.7.